go/v1/statement.go

```diff
@@ -17,7 +21,7 @@ var (
 )
 
 func (s *Statement) Validate() error {
-	if s.GetType() != StatementTypeUri {
+	if !strings.HasPrefix(s.GetType(), StatementTypeUriPrefix) {
```

Comment:
I think this is actually a little too permissive. I would rather check the two specific supported versions.